Doc/extending/extending.rst

```diff
@@ -127,10 +127,10 @@ Intermezzo: Errors and Exceptions
 
 An important convention throughout the Python interpreter is the following: when
 a function fails, it should set an exception condition and return an error value
-(usually a ``NULL`` pointer).  Exceptions are stored in a static global variable
-inside the interpreter; if this variable is ``NULL`` no exception has occurred.  A
-second global variable stores the "associated value" of the exception (the
-second argument to :keyword:`raise`).  A third variable contains the stack
+(usually a ``NULL`` pointer).  Exception information is stored in 3 static
+global variables inside the interpreter.  The first is the exception type,
+or ``NULL`` if no exception has occurred.  The second is the exception
+instance.  The third variable contains the stack
```

Comment:
I'd write "three" in words instead of "3".

Technically these are not static global variables, they are members of the thread state nowadays.

If no exception occurred all three are none.

I think there are edge cases where the exception type is set but the instance is still NULL.

I also think the traceback may be NULL even though an exception is set (probably it's NULL until the exception bubbles into a Python frame.

Reply:
'three', 'members', and 'all' done.  Otherwise I'm going with the reference to sys.exc_info.